Cargo.toml

```diff
@@ -35,3 +35,4 @@ glsl-to-spirv = "0.1"
 log = "0.4"
 png = "0.15"
 winit = "0.20.0-alpha4"
+image = "0.22"
```

Comment:
would it be possible to use `png` images for the cubemap? Keeping it only to `png` dependency would mean our test builds are faster and breaking less often because of unrelated stuff in `image`, which is bigger.

Reply:
Sure, will update